engine/src/main/java/org/terasology/logic/console/commands/CoreCommands.java

```diff
@@ -506,24 +526,91 @@ public String spawnBlock(@Sender EntityRef sender, @CommandParam("blockName") St
         spawnPos.add(offset);
 
         BlockFamily block = blockManager.getBlockFamily(blockName);
-        if (block == null) {
-            return "";
-        }
 
         BlockItemFactory blockItemFactory = new BlockItemFactory(entityManager);
         EntityRef blockItem = blockItemFactory.newInstance(block);
 
         blockItem.send(new DropItemEvent(spawnPos));
         return "Spawned block.";
     }
+    @Command(shortDescription = "Drops many blocks as per value choosen by the player", helpText = "the particular block can be seleted which the player wants to drop"
```

Comment:
Help text here is a bit weird and has a typo. Also doesn't touch the `value` param (doesn't need a full independent description or anything but could be something like "You can indicate which block to drop with the first parameter and how many with the second")
  
Could the `value` param be optional with a default value of maybe 100? Then the player could just use `bulkDrop dirt`. And on a similar note should `value` be capped? Have you tried to see what a reasonable upper bound would be? If 2000 blocks lags even a good system we might as well make maybe 10000 the max and return a message to the player if a higher number was requested.